applications/RANSApplication/custom_utilities/rans_variable_utilities.cpp

```diff
@@ -304,6 +304,44 @@ std::tuple<double, double> CalculateTransientVariableConvergence(
     KRATOS_CATCH("");
 }
 
+template <class TContainerType>
+void SetContainerConstitutiveLaws(TContainerType& rContainer)
```

Comment:
Which is the intended use of this method? Shouldn't the constitutive law pointer be assigned when reading the materials.json?

Reply:
It is assigned, but to an element private member which is not accessible from outside. In RANS I create duplicate model parts having the same geometry with different elements and conditions. I want to reuse the same CL to reduce memory consumption so, whenever I create a duplicated model part, if `CONSTITUTIVE_LAW` variable is not assigned with a pointer in the geometry data value container this method will assign it based on the `Properties`. Some RANS processes also need to access CL, so I need to keep it in the geometry data value container :/

Reply:
OK. Clarified! Thanks for the explanation!